test-ts/manager.test.ts

```diff
@@ -77,20 +124,20 @@ describe("Manager", () => {
       groups.push(group);
       groupAddresses.push(group.address);
     }
-
-    lockedGold = await hre.kit.contracts.getLockedGold();
     for (let i = 0; i < 11; i++) {
       if (i == 1) {
         // For groups[1] we register an extra validator so it has a higher voting limit.
         await registerValidatorGroup(groups[i], 2);
         const [validator, validatorWallet] = await randomSigner(parseUnits("11000"));
-        await registerValidator(groups[i], validator, validatorWallet);
+        await registerValidatorAndAddToGroupMembers(groups[i], validator, validatorWallet);
       } else {
         await registerValidatorGroup(groups[i], 1);
       }
       const [validator, validatorWallet] = await randomSigner(parseUnits("11000"));
-      await registerValidator(groups[i], validator, validatorWallet);
+      await registerValidatorAndAddToGroupMembers(groups[i], validator, validatorWallet);
     }
+
+    await electMinimumNumberOfValidators(groups, voter); // first 10 groups
```

Comment:
I'm wondering if not to pass number of groups (as named constant) into this function. Later on it might be a bit clearer why groups[10] is being accessed (10 could be replaced with named constant)

Reply:
Or could minimum number of validators be obtained from some smart contract ?